src/services/sezion/index.js

```diff
@@ -0,0 +1,72 @@
+const httpClient = require('./httpClient');
+const { flatten } = require('lodash');
+const utils = require('./utils');
+const { template } = require('./defaults');
+
+const SezionApi = {
+    // Templates
+    getTemplates: () => new Promise((resolve) => {
+        httpClient.request((sezionAPI) => {
+            sezionAPI.Template_List((err, result) => resolve(result));
+        });
+    }),
+    createTemplate: ({ name, description, templateObjectsList }) => new Promise((resolve, reject) => {
+        // Set correct template structure
+        const inputScriptsConfig = utils.createTemplateFromList(templateObjectsList);
+        const templateData = Object.assign({}, template, {
+            name,
+            description,
+            inputScripts: utils.getTemplateShape(inputScriptsConfig),
+            videoInputs: utils.getObjectsTypeCountFromList(templateObjectsList),
+        });
+
+        httpClient.request((sezionAPI) => {
+            sezionAPI.Template_New(
+                templateData,
+                (err, result) => {
+                    err && reject(JSON.stringify(err));
+                    resolve(result)
+                }
+            );
+        });
+    }),
+
+    // Videos
+    newVideo: (videoData, templateId) => new Promise((resolve) => {
+        httpClient.request((sezionAPI) => {
+            sezionAPI.Template_Video_New(
+                templateId,
+                videoData,
+                (err, result) => resolve(result)
+            );
+        });
+    }),
+    getVideosFromTemplateId: (templateId) => new Promise((resolve) => {
+        httpClient.request((sezionAPI) => {
+            sezionAPI.Template_Video_List(templateId, (err, result) => resolve(result));
+        });
+    }),
+    getVideos: () => new Promise((resolve) => {
+        SezionApi.getTemplates().then((templates) => {
+            const videosRequests = templates.map(({ id }) => SezionApi.getVideosFromTemplateId(id));
+            Promise.all(videosRequests).then((videos) => resolve(flatten(videos)));
```

Comment:
There is no specific api to get all the videos so we have to get the videos created with each template and put them together.